### DISCUSSION TITLE
无法连接

### DISCUSSION THREAD

**Discussion by Xraytest** [1715173329/UnblockNeteaseMusic#914] [Q&A]
使用代理后（HTTP/HTTPS都试了），客户端无法连接到网络

**Comment by 1715173329** [comment]
> https://httpd.apache.org/docs/2.4/getting-started.html#:~:text=Troubleshooting%20any%20problem%20without%20the,you%20how%20to%20fix%20it.

Troubleshooting any problem without the error log is like driving with your eyes closed.
